mpcs/benches/basefold.rs

```diff
@@ -73,12 +73,12 @@ fn bench_commit_open_verify_goldilocks<Pcs: PolynomialCommitmentScheme<E>>(
         let point = get_point_from_challenge(num_vars, &mut transcript);
         let eval = poly.evaluate(point.as_slice());
         transcript.append_field_element_ext(&eval);
-        let transcript_for_bench = transcript.clone();
+        let transcript_for_bench = transcript;
         let proof = Pcs::open(&pp, &poly, &comm, &point, &eval, &mut transcript).unwrap();
 
         group.bench_function(BenchmarkId::new("open", format!("{}", num_vars)), |b| {
             b.iter_batched(
-                || transcript_for_bench.clone(),
```

Comment:
We remove the need to clone here in https://github.com/scroll-tech/ceno/pull/770

That PR is somewhat independent of yours, and we can merge it first, making the diff here smaller.